src/Settings/Builders/Ghc.hs

```diff
@@ -15,6 +19,23 @@ ghcBuilderArgs = (builder (Ghc CompileHs) ||^ builder (Ghc LinkHs)) ? do
             , append =<< getInputs
             , arg "-o", arg =<< getOutput ]
 
+ghcCbuilderArgs :: Args
+ghcCbuilderArgs =
+  builder (Ghc GHCCompileC) ? do
+    confCc  <- getSettingList . ConfCcArgs =<< getStage
+    confCpp <- getSettingList . ConfCppArgs =<< getStage
+    cppArgs <- getPkgDataList CppArgs
+    mconcat [ arg "-Wall"
+            , includeGhcArgs
+            , packageGhcArgs
```

Comment:
Shouldn't this include `commonGhcArgs` as well? On my machine I noticed that this patch resulted in linker errors apparently due to missing `-fPIC` flags during the RTS build. Presumably this is the reason.